player/tests.py

```diff
@@ -0,0 +1,93 @@
+from django.contrib.auth import get_user_model
+from .models import Player, BallGame
+from django.core.exceptions import ValidationError
+import pytest
+
+
+@pytest.mark.django_db
+class TestPlayerModel:
+
+    @pytest.fixture
+    def user(self):
+        return get_user_model().objects.create_user(
+            username='testuser',
+            password='testpass'
+        )
+
+    @pytest.fixture
+    def saved_player(self, user):
+        player = Player.objects.create(
+            user=user,
+            birth_date='1990-01-01',
+            favorite_ball_game=BallGame.Soccer
+        )
+        return player
+
+    def test_get_player(self, saved_player):
+        player = Player.objects.get(pk=saved_player.pk)
+        assert player == saved_player
+
+    def test_create_player(self, saved_player, user):
+        player = saved_player
+        assert player.user == user
+        assert player.birth_date == '1990-01-01'
+        assert player.favorite_ball_game == BallGame.Soccer
+
+    def test_update_player(self, saved_player):
+        player = saved_player
+        player.favorite_ball_game = BallGame.Basketball
+        player.save()
+        updated_player = Player.objects.get(pk=player.pk)
+        assert updated_player.favorite_ball_game == BallGame.Basketball
+
+    def test_delete_player(self, saved_player):
+        player = saved_player
+        player.delete()
+        assert not Player.objects.filter(pk=player.pk).exists()
```

Comment:
I suggest this approach as you're using it to verify that the player does exist when it should.
```suggestion
        with pytest.raises(Player.DoesNotExist):
            Player.objects.get(pk=saved_player.pk)
```